packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -155,7 +155,10 @@ export default class ListControl extends React.Component {
     const { onChange } = this.props;
     const oldValue = this.state.value;
     const newValue = e.target.value;
-    const listValue = e.target.value.split(',');
+    let listValue = [];
+    if (e.target.value.trim().length > 0) {
```

Comment:
why not reuse newValue variable?

Reply:
Also, seems that it can be simplified to
```js
const newValue = e.target.value.trim()
const listValue = newValue ? newValue.split(',') : []
```
because newValue should always be a string

Reply:
Thanks for the comment. Changes have been applied as suggested.